test_regress/t/t_union_hard.out

```diff
@@ -0,0 +1,6 @@
+%Error: t/t_union_hard.v:11:21: Hard packed union members must have equal size (IEEE 1800-2023 7.3.1)
```

Comment:
Please rename test to t_union_hard_bad.* as we use _bad to indicate tests are IEEE-illegal.